examples/email-mcp/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@xava-labs/examples-email-mcp",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "wrangler dev",
+    "build": "tsc -p tsconfig.json",
+    "deploy": "wrangler deploy",
+    "d1:apply": "wrangler d1 migrations apply email_db --local",
+    "test": "vitest"
+  },
+  "devDependencies": {
+    "@cloudflare/vitest-pool-workers": "^0.6.31",
```

Comment:
We recently added catalog: versions at the root - please update to use root workspace catalog deps